test/image/mocks/zindex_basic.json

```diff
@@ -0,0 +1,53 @@
+{
+    "data": [
+        {
+            "x": [2, 3, 4, 5],
+            "y": [4, 3, 9, 5],
+            "name": "blue",
+            "type": "scatter",
+            "marker": {"size": 20},
+            "line": {"width": 6},
+            "zindex": 5
+        },
+        {
+            "x": [2, 3, 4, 5, 6],
+            "y": [2, 5, 4, 6, 4],
+            "name": "orange",
+            "type": "scatter",
+            "marker": {"size": 20},
+            "line": {"width": 6},
+            "zindex": 4
+        },
+        {
+            "x": [3, 4, 5, 6, 7],
+            "y": [1, 8, 6, 8, 5],
+            "name": "green",
+            "type": "scatter",
+            "marker": {"size": 20},
+            "line": {"width": 6}
+        },
+        {
+            "x": [4, 5, 6, 7],
+            "y": [2, 8, 2, 7],
+            "name": "red",
+            "type": "scatter",
+            "marker": {"size": 20},
+            "line": {"width": 6},
+            "zindex": 300
+        },
+        {
+            "x": [2, 3, 4, 5, 6, 7],
+            "y": [4, 2, 6, 1, 6, 9],
+            "name": "purple",
+            "type": "bar",
+            "zindex": 6
+        }
+    ],
+    "layout": {
```

Comment:
Also here let's provide `width` and `height` to make it smaller as it looks good.